packages/slate-react/src/components/editable.tsx

```diff
@@ -1326,6 +1326,9 @@ const defaultScrollSelectionIntoView = (
   editor: ReactEditor,
   domRange: DOMRange
 ) => {
+  if (editor.selection?.anchor?.path[0] !== editor.selection?.focus?.path[0]) {
```

Comment:
Not sure if the check is for an expanded range, if so `Range.isExpanded` should be used. If not, a variable should be extracted so that the if is a bit more readable.

Reply:
I forgot that we have helper methods.